integ/src/eks_provider.rs

```diff
@@ -270,7 +270,7 @@ impl SecurityGroupType {
             SecurityGroupType::ClusterShared => "ClusterSharedNodeSecurityGroup",
```

Comment:
This might not be generic enough. The security group used for IPv6 cluster is named like `eksctl-brupop-ipv6-nodegroup-ng-brupop-ipv6-remoteAccess`, even though it has something like "ClusterSharedNodeSecurityGroup". Could you please test in IPv6 cluster as well?

Reply:
I think `format!("*{}*{}*", cluster_name, sg)` still can handle ipv6 security group format. I'll test it with IPv6 cluster anyway. :)